Dockerfile

```
@@ -44,42 +37,49 @@ RUN apt-get -y --no-install-recommends install \
 # install R packages from CRAN
 RUN install2.r \
 	BiocManager \
-  cDriver \
-  clonevol \
   data.table \
   devtools \
   fishplot \
   flextable \
   future \
+  ggplot2\ 
   ggpubr \
   ggthemes \
-  immunedeconv \
-  msigdbr \
+  gridBase \
+  gridExtra \
+  igraph \
 	optparse \
+  packcircles \
 	pheatmap \
   plyr \
   purrr \
 	RColorBrewer \
+  R.utils \
   rlist \
-	survival \
-  survMisc \
-  survminer \
   tidytext \
+  tidyverse \
   vroom
 
+
 # install R packages from Bioconductor 
 RUN ./install_bioc.r \
-  ComplexHeatmap \
-  GSVA 
+  ComplexHeatmap
 
 # install R packages from GitHub
 
 # Maftools
 RUN ./install_github.r \
+  cDriver \
```

```suggestion
RUN ./install_github.r \
  cDriver 'hanasusak/cDriver' --ref 'aa466ae92095f877a1f68ee7d2e6136cba9b02ea'
```

see ex here: 

https://github.com/diskin-lab-chop/OpenPBTA-germline/blob/75055b69e583d7e4f8ee0e3e424e8a653574fff8/Dockerfile#L76C36-L76C36

---

can you do this for all github ones

---

done that